test/distributed/test_local_dataset.py

```diff
@@ -0,0 +1,61 @@
+import os.path as osp
+
+import torch
+import torch.nn.functional as F
+from tqdm import tqdm
+
+import torch_geometric.transforms as T
+from torch_geometric.datasets import OGB_MAG
+from torch_geometric.distributed import LocalDataset
+
+
+def test_local_dataset():
+
+    path = osp.join(osp.dirname(osp.realpath(__file__)), './ogbn-mags')
+    transform = T.ToUndirected(merge=True)
+    dataset = OGB_MAG(path, preprocess='metapath2vec', transform=transform)
```

Comment:
You cannot use `OGB_MAG` in a test. I suggest to use a small custom dataset, or use `FakeHeteroDataset` for this.

Reply:
Sure. Will change it. thanks.